static/css/page-barcodescanner.less

```diff
@@ -0,0 +1,145 @@
+@import (less) "less/index.less";
+
+// Import all base styles
+@import (less) "base/index.less";
+
+// layout rules
+@import (less) "layout/index.less";
+
+// Components (modules) in order of appearance on page
+@import (less) "components/header.less";
+@import (less) "components/carousel.less";
+@import (less) "components/home.less";
+@import (less) "components/footer.less";
+
+@highlight-color: @blue;
+
+@keyframes pulse {
+  0% { opacity: 0; }
+  100% { opacity: .95; }
+}
+
+@keyframes slideUp {
+  0% { transform: translateY(50%); opacity: .5; }
+  100% { transform: translateY(0); opacity: 1; }
+}
+
+@keyframes shiftRight {
+  0% { transform: translateX(-100%); }
+  100% { transform: translateX(0); }
+}
+
+#interactive.viewport {
+  position: relative;
+  width: 100%;
+
+  video, canvas {
+    width: 100%;
+    height: 100vh;
+    object-fit: cover;
+  }
+
+  canvas {
+    position: absolute;
+    top: 0;
+    left:0;
+  }
+}
+
+#page-barcodescanner {
+  position: relative;
+}
+
+#result-strip {
+  font-family: @lucida_sans_serif-1;
+  display: flex;
+  overflow-x: auto;
+  overflow-y: hidden;
+  position: absolute;
+  bottom: 0;
+  background: fade(@white, 90%);
+  width: 100%;
+  padding: 10px;
+
+  .empty {
+    display: none;
+    &:first-child:last-child { display: flex; }
+    width: 80vw;
+    max-width: 300px;
+    height: 80px;
+    border: 1px dashed;
+    justify-content: center;
+    align-items: center;
+    flex-shrink: 0;
+    border-radius: 4px;
+  }
+}
+
+a.lazy-book-card {
```

Comment:
should we move these classes inside the the `#page-barcodescanner` to prevent leakage?